### PR TITLE
604--tab-completion

### PR DESCRIPTION
**Pull request by adschwartz**
Update with new brew based bash installation

### CODE REVIEW

File: docs/guides/adding-tab-completion.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ This page was generated by referencing the kubectl docs:
 
 -->
 
-[The Kurtosis CLI](/cli) supports tab completion for `bash`, `zsh`, and `fish` via the `kurtosis completion SHELL` command (e.g. `kurtosis completion bash`). `source`ing the output of the command will enable tab-completion, and adding the `source` command to your shell config file will enable it across shells. 
-
-Once tab completion is installed, you will be able to:
+[The Kurtosis CLI](/cli) supports tab completion for `bash`, `zsh`, and `fish`. With tab completion is installed, you will be able to:
```
Comment (h4ck3rk3y) [inline docs/guides/adding-tab-completion.md:16]: There is an extra is

s/With tab completion is installed/With tab completion installed/g

File: docs/guides/adding-tab-completion.md [hunk 2]
```diff
@@ -115,3 +115,23 @@ Once tab completion is installed, you will be able to:
 1. Reload your shell
 
 </details>
+
+<details>
+<summary>
+manual installation
+</summary>
```
Comment (h4ck3rk3y) [inline docs/guides/adding-tab-completion.md:122]: we seem to have individual instructions up in the page, is this a dupe?

**Review by h4ck3rk3y — APPROVED** [review]

File: docs/guides/adding-tab-completion.md (hunk 1 above)
Comment (adschwartz) [reply docs/guides/adding-tab-completion.md]: Thanks. Fixed

File: docs/guides/adding-tab-completion.md (hunk 2 above)
Comment (adschwartz) [reply docs/guides/adding-tab-completion.md:122]: No actually I moved those instructions from the top of the page down here to clarify that they are only necessary if you are installing manually. I added a blurp to help clarify that based on your question here.

**Pull request merged**